include/irr/core/alloc/LinearAddressAllocator.h

```diff
@@ -45,6 +45,12 @@ class LinearAddressAllocator : public AddressAllocatorBase<LinearAddressAllocato
             bufferSize = newBuffSz-Base::alignOffset;
             std::swap(cursor,other.cursor);
         }
+        template<typename... Args>
+        LinearAddressAllocator(_size_type newBuffSz, const LinearAddressAllocator& other, Args&&... args) :
+            Base(other,std::forward<Args>(args)...), bufferSize(invalid_address), cursor(invalid_address)
```

Comment:
you need to compute the cursor properly (copy it)

Reply:
erm, thanks, seems like i copy-pasted and forgot about this